lib/cmd/lint.js

```diff
@@ -156,7 +222,7 @@ function checkPublicUrl(url, concurrency) {
     .flatMap(({ uri, prefix }) => {
       const pageURL = prefixes.uriToUrl(prefix, uri);
 
-      return h([h.of({ type: 'success', message: url }), checkPage(pageURL, prefix, concurrency)]).merge();
+      return h([h.of({ type: 'success', message: url }), checkPage(`${pageURL}.html`, prefix, concurrency)]).merge();
```

Comment:
should pass `${ext}` here instead of hard-coding `.html`?

Reply:
Nope! Public urls always point to page html

Reply:
erm, right then!